ios/FluentUI.Demo/FluentUI.Demo/Demos/ListVnextDemoController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class ListVnextDemoController: DemoController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let sections = TableViewCellSampleData.sections
+        var section: TableViewCellSampleData.Section
+        var cell: TableViewCellSampleData.Item
+        var indexPath = IndexPath(row: 0, section: 0)
+
+        var list: MSFListVnext
+        var listCell: MSFListVnextCellData
+        var listSection: MSFListVnextSectionData
+        var listData: [MSFListVnextSectionData] = []
+        let iconStyle = MSFListIconVnextStyle.none
+
+        for sectionIndex in 0...sections.count - 1 {
+            section = sections[sectionIndex]
+
+            listSection = MSFListVnextSectionData()
+            listSection.title = section.title
+            listSection.cells = []
+            for rowIndex in 0...TableViewCellSampleData.numberOfItemsInSection - 1 {
+                cell = section.item
+                listCell = MSFListVnextCellData()
+                listCell.title = cell.text1
+                listCell.subtitle = cell.text2
+                listCell.leadingIcon = UIImage(named: cell.image)
+                listCell.accessoryType = accessoryType(for: rowIndex)
+                listCell.onTapAction = {
+                    indexPath.row = rowIndex
+                    indexPath.section = sectionIndex
+                    self.showAlertForCellTapped(indexPath: indexPath)
+                }
+                listSection.cells.append(listCell)
+            }
+            listSection.layoutType = updateLayout(subtitle: listSection.cells[0].subtitle)
+             listData.append(listSection)
+        }
+        list = MSFListVnext(sections: listData, iconStyle: iconStyle)
+
+        let listView = list.view
+        listView.translatesAutoresizingMaskIntoConstraints = false
+
+        let demoControllerView: UIView = self.view
+        demoControllerView.addSubview(listView)
+
+        NSLayoutConstraint.activate([demoControllerView.safeAreaLayoutGuide.topAnchor.constraint(equalTo: listView.topAnchor),
```

Comment:
If you are using a custom constraint it would a good idea to not inherit from `DemoController` and just use `UIViewController`